src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  padding: 0;
+  
+  --activation-color: #00ACDC;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  width: 100vw;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.logo {
+  height: 40px;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0 40px;
+  padding: 0;
+}
+
+.nav__link {
+  display: inline-block;
+  position: relative;
+  font-size: 12px;
+  margin: 0 10px;
+  line-height: 60px;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Use padding instead